tests/base_test.py

```diff
@@ -36,7 +33,9 @@ def iter_verbs(self, c):
         for verb in ['get', 'head', 'options']:
             yield getattr(c, verb)
 
-    def iter_responses(self, path, verbs=['get', 'head', 'options'], **kwargs):
+    def iter_responses(self, path, verbs=None, **kwargs):
+        if verbs is None:
```

Comment:
Sure, I can revert this next time I'm at the keyboard. Although please read this on mutable default parameters. It's something I only recently learned and it might be useful here too. Thank you! 

https://docs.python-guide.org/writing/gotchas/